language-design.md

```diff
@@ -80,4 +84,4 @@ you do so? If not, why not?
 
 **Response**
 
----
+I think these two experiences are a bit at odds, but that doesn't erase the validity of either one. It is true that a more natural looking language is easier to read and initially parse, but as Cook says, it is still an artificial language. I might still choose to incorporate natural language into the design of a DSL because I think, as Pavlus argued, it can be easier to parse in the user's brain if there is more natural language for odd syntax. That being said, just because some things might resemble natural language or a more informal style like psuedo-code, I would be very careful with where and how I did that so that it still maintains the principles of good design that I've discussed in other questions, like predictability and consistency. For example, I wouldn't want to use a natural language word that has multiple meanings and uses, otherwise it might be confusing or non-intuitive what the case of the DSL would be.
```

Comment:
I would also add in that another important principle of good design is conciseness. I don't believe this is a principle involved from the reading about how to create a good API, but seeing things like how to print a string in Java makes it clear that more is not always better. However, neither are one-liners because they are so dense with content that it's difficult to parse; so introducing more natural language -- as you have mentioned you'd like to do -- will hopefully bring more clarity at the cost of brevity.